src/main/resources/assets/tconstruct/book/materials_and_you/sections/introduction.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "name": "advertisements",
```

Comment:
I was envisioning book recipes with this, the whole point of the advertisements was to tell the player how to craft the next books. I'd suggest moving this page into the further reading section instead.